crates/sui-bridge-indexer/src/sui_worker.rs

```diff
@@ -32,7 +32,7 @@ pub struct SuiBridgeWorker {
 impl SuiBridgeWorker {
     pub fn new(
         bridge_object_ids: Vec<ObjectID>,
-        db_url: String,
+        db_url: Option<String>,
```

Comment:
don't pass the Option in, check it in the callsite?

Reply:
I kinda suck at rust, but I wasn't able to easily unwrap the string at the callsite because of the borrow checker (i had to unwrap in multiple callsites):
```
--> crates/sui-bridge-indexer/src/main.rs:89:18
    |
89  |     let db_url = config.db_url.unwrap();
    |                  ^^^^^^^^^^^^^ -------- `config.db_url` moved due to this method call
    |                  |
    |                  help: consider calling `.as_ref()` or `.as_mut()` to borrow the type's contents
    |                  move occurs because `config.db_url` has type `Option<std::string::String>`, which does not implement the `Copy` trait
    |
note: `Option::<T>::unwrap` takes ownership of the receiver `self`, which moves `config.db_url`
   --> /Users/johnmartin/.rustup/toolchains/1.75-aarch64-apple-darwin/lib/rustlib/src/rust/library/core/src/option.rs:928:25
   ```